source/appModules/hxmail.py

```diff
@@ -12,13 +12,32 @@
 
 class MailWordDocumentTreeInterceptor(WordDocument.treeInterceptorClass):
 
-	def _get_isAlive(self):
-		return super(MailWordDocumentTreeInterceptor,self).isAlive and self.rootNVDAObject.shouldCreateTreeInterceptor
+	_wasInReadingPane: bool = False
+
+	def event_treeInterceptor_gainFocus(self):
+		isInReadingPane = self.rootNVDAObject.isInReadingPane

```

Comment:
This assumes that `rootNVDAObject` will always be a `MailWordDocument`. If I'm understanding this right, while unlikely, a developer may set this as the `TreeInterceptor` on a different object, resulting in it being constructed with a different type as the `rootNVDAObject`.

This took me quite some time to determine, typing would help, maybe something like this at class level:
``` python
rootNVDAObject: MailWordDocument

def __init__(self, obj:MailWordDocument):
		if not isinstance(obj, MailWordDocument):
				log.error("MailWordDocumentTreeInterceptor depends on rootNVDAObject being a MailWordDocument see event_treeInterceptor_gainFocus")
		super().__init(obj)
```